stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -204,10 +204,13 @@ spec:
             printf "\nsentinel myid $(echo $HOSTNAME | openssl sha1 | awk '{ print $2 }')" >> /opt/bitnami/redis-sentinel/etc/sentinel.conf
             {{- end }}
           fi
-          echo "Getting information about current running sentinels"
-          # Get information from existing sentinels
-          existing_sentinels=$(timeout -s 9 {{ .Values.sentinel.initialCheckTimeout }} redis-cli --raw -h {{ template "redis.fullname" . }} -a "$REDIS_PASSWORD" -p {{ .Values.sentinel.service.sentinelPort }} SENTINEL sentinels {{ .Values.sentinel.masterSet }})
-          echo "$existing_sentinels" | awk -f /health/parse_sentinels.awk | tee -a  /opt/bitnami/redis-sentinel/etc/sentinel.conf
+          if grep -q known-sentinel /opt/bitnami/redis-sentinel/etc/sentinel.conf; then
+            echo "Getting information about current running sentinels"
+            # Get information from existing sentinels (if pod was freshly created only)
+            existing_sentinels=$(timeout -s 9 {{ .Values.sentinel.initialCheckTimeout }} redis-cli --raw -h {{ template "redis.fullname" . }} -a "$REDIS_PASSWORD" -p {{ .Values.sentinel.service.sentinelPort }} SENTINEL sentinels {{ .Values.sentinel.masterSet }})
+            echo "$existing_sentinels" | awk -f /health/parse_sentinels.awk | tee -a  /opt/bitnami/redis-sentinel/etc/sentinel.conf
+          fi
+          bash -c "sleep 120; $(timeout -s 9 {{ .Values.sentinel.initialCheckTimeout }} redis-cli --raw --no-auth-warning -h localhost -a \"$REDIS_PASSWORD\" -p {{ .Values.sentinel.service.sentinelPort }} SENTINEL sentinels {{ .Values.sentinel.masterSet }} | awk -f /health/parse_sentinels_reset.awk) echo; redis-cli --raw --no-auth-warning -h localhost -a \"$REDIS_PASSWORD\" -p {{.Values.sentinel.porti }} SENTINEL reset {{ .Values.sentinel.masterSet }}" &
```

Comment:
If anyone like me is looking for this fix based on Sentinel issues in this chart version - notice the typo on this line: ".Values.sentinel.porti" -> ".Values.sentinel.port"

Reply:
oh thanks - that's in fact a typo. Unfortunately noone of the maintainer wants to merge it finnaly anymore: https://github.com/helm/charts/pull/21168#issuecomment-611970849
I'll try to ropen this PR with corrected port variable name but I'm afraid, the whole chart is really depracated.

Reply:
this repo will have been closed the day after tomorrow - it makes no sense to update my PR, but thanks for mention this error !